include/irr/core/math/aabb2dSIMDIntBase.h

```diff
@@ -216,9 +175,80 @@ class aabbSIMD<core::vector2df_SIMD> : public aabbSIMDBase<aabbSIMD, core::vecto
 
 };
 
+template<typename VECTOR_TYPE>
+class aabb2dSIMDInt : public aabbSIMDBase<aabb2dSIMDInt, VECTOR_TYPE>
+{
+	static_assert(
+		std::is_same<VECTOR_TYPE, core::vector2di32_SIMD>::value ||
+		std::is_same<VECTOR_TYPE, core::vector2du32_SIMD>::value,
+		"assertion failed: cannot use this type");
+
+public:
+	inline aabb2dSIMDInt()
+	{
+		if constexpr (std::is_same<VECTOR_TYPE, core::vector2di32_SIMD>::value)
+			internalBoxRepresentation = core::vector4di32_SIMD(INT_MIN);
```

Comment:
you need `(INT_MAX,INT_MAX,INT_MIN,INT_MIN)` I told you that beacuse of the ranges you won't be able to use the same "one SSE op" trick as for the `vectorSIMDf` based solution

Reply:
wtf i did xd

Reply:
?

Reply:
nvm